### PR TITLE
all function added as an instance method to jax frontend

### PR DESCRIPTION
**Pull request by FaikYY**

### CODE REVIEW

**Comment by FaikYY** [comment]
#7345

**Comment by FaikYY** [comment]
@rahulsnkr kindly review my pull request and let me know if further updates are needed. Thanks!

File: ivy/functional/frontends/jax/devicearray.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def at(self):
     # Instance Methods #
     # ---------------- #
 
+    def __all__(self, other):
```
Comment (rahulsnkr) [inline ivy/functional/frontends/jax/devicearray.py:26]: @FaikYY thanks for your contribution! Could you please remove the changes to the files in `.idea` folder from your commit? Also, would be great if you could also write a test for your implementation. You can have a look at the [docs ](https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html) and the tests for other  [jax instance methods](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py) to get an idea of how to test your function.

**Review by rahulsnkr — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/devicearray.py (hunk 1 above)
Comment (FaikYY) [reply ivy/functional/frontends/jax/devicearray.py:26]: Thanks a lot for your feedback! I will remove the changes from .idea file and write a test.

**Comment by FaikYY** [comment]
@rahulsnkr I have removed the changes made in .idea folder and created the test. It passed all four of them succesfully. Please do let me know if further changes required. Thanks!

**Comment by rahulsnkr** [comment]
@FaikYY I see that `.idea/runConfigurations/pytest_for_all__jax_.xml` is still included in the commit, could you remove it please?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py [hunk 2]
```diff
@@ -260,6 +260,28 @@ def test_jax_special_rpow(
     )
 
 
+# __all__
+@handle_frontend_test(
+    fn_tree="jax.lax.add",  # dummy fn_tree
```
Comment (rahulsnkr) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py:265]: this test should be for the function you're implementing instead of `jax.lax.add`

**Review by rahulsnkr — CHANGES_REQUESTED** [review]

**Comment by FaikYY** [comment]
@rahulsnkr I hope I was able to resolve the issues you have mentioned earlier. If not please guide me through so that I can obtain the necessary knowledge and resolve the remaining problems. Thanks!

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py [hunk 3]
```diff
@@ -260,6 +260,28 @@ def test_jax_special_rpow(
     )
 
 
+# __all__
+@handle_frontend_test(
+    fn_tree="numpy.ndarray",  # dummy fn_tree
```
Comment (rahulsnkr) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py:265]: This should refer to the function you're implementing instead of `numpy.ndarray`. Refer to the implementations of other `jax devicearray` functions to get an idea. 

Also, I see that the commit still contains a file from the `.idea` folder. You can check online on how to add only specific files to a commit. Maybe [this link](https://www.nobledesktop.com/learn/git/stage-commit-files) would be helpful.

**Review by rahulsnkr — CHANGES_REQUESTED** [review]

**Comment by FaikYY** [comment]
I have updated the dummy tree section and added only the file I am supposed to be updating.

**Comment by FaikYY** [comment]
@rahulsnkr

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by rahulsnkr** [comment]
@FaikYY I see that this file is still present in the commit `.idea/runConfigurations/pytest_for_all__jax_.xml`
image

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by FaikYY** [comment]
Okay, I am working on it.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
